views/js/previewer/plugins/content/scoringHighlighter.js

```diff
@@ -38,11 +38,15 @@ define([
          */
         init() {
             const testRunner = this.getTestRunner();
+
             this.selection = window.getSelection();
-            
+
             this.isEraserOn = false;
+            this.isHighlighterOn = false;
```

Comment:
Should it be value instead of boolean? We are planning to use multiple colors in future

Reply:
This var was unused, removed it